easybuild/easyblocks/p/pgi.py

```diff
@@ -58,16 +61,88 @@ def extra_options():
     def __init__(self, *args, **kwargs):
         """Easyblock constructor, define custom class variables specific to PGI."""
         super(EB_PGI, self).__init__(*args, **kwargs)
-        if not self.cfg['license_file']:
-            self.cfg['license_file'] = 'UNKNOWN'
+
+        self.license_file = 'UNKNOWN'
+        self.license_env_var = 'UNKNOWN' # Probably not really necessary for PGI
+
         self.install_subdir = os.path.join('linux86-64', self.version)
 
     def configure_step(self):
         """
-        Dummy configure method, just a license check
+        Handle license file. It mimics intelbase.py behaviour. It should be merged into the framekwork
+        at some point
```

Comment:
@damianam: here you go https://github.com/hpcugent/easybuild-framework/pull/1633 :)